docs/misc/defaultlabels.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 1
+---
+
+# Default Labels
+
+Default labels allow team members to pre-load a number of labels in order to simplify the labelling process.
+
+To setup default labels, open **Curate** for the relevant project.
+
+![Default Label Pop-Up]
+
+Click the **Set default labels** button and a pop-up should appear.
+By typing in the **New label** box and clicking **+** you can add as many default labels as desired.
+You can then decide, using the check boxes, whether to restrict collaborators (including all active users) to just these labels and whether to allow multiple labels per image.
+Once you have the settings desired, click the **Confirm** button to apply the changes.
+
+_Please Note: In the current version, default labels apply universally to Curate and Annotate._
```

Comment:
Change to:
`_Please note: default labels apply to both Curate and Annotate_`?
Not sure we want to imply that that is going to change?